src/components/ScreenWrapper/propTypes.js

```diff
@@ -1,5 +1,4 @@
 import PropTypes from 'prop-types';
-import {windowDimensionsPropTypes} from '../withWindowDimensions';
```

Comment:
We use withWindowDimensions in the ScreenWrapper and we access the windowHeight too, so we shouldn't remove this. https://github.com/Expensify/App/blob/8808ed68f5049557217c4cf8005b2780ed409913/src/components/ScreenWrapper/BaseScreenWrapper.js#L128

Reply:
it's just moved because we use these props in index.js and index.native.js and they are required (so in this case we'd need to use withDimensions in index/index.native.js)
https://github.com/Expensify/App/blob/8808ed68f5049557217c4cf8005b2780ed409913/src/components/ScreenWrapper/BaseScreenWrapper.js#L25-L29

Reply:
Ok, but BaseScreenWrapper should import all it's prop types from ScreenWrapper/propTypes.